pom.xml

```diff
@@ -623,71 +623,71 @@
         <james.protocols.groupId>${james.groupId}.protocols</james.protocols.groupId>
         <activemq.version>6.1.6</activemq.version>
         <apache-mime4j.version>0.8.12</apache-mime4j.version>
-        <apache.openjpa.version>4.0.0</apache.openjpa.version>
+        <apache.openjpa.version>4.1.1</apache.openjpa.version>
```

Comment:
Do not forget to update JPA version in the JPA docker images too

Reply:
Not been updated for a while clearly... Ok done